lib/stars.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'movie.dart';
+
+class Stars extends StatelessWidget {
+  final Movie movie;
+
+  static const ratingIcon = Icons.star;
+
+  const Stars(
```

Comment:
Review the parameter that you are sending to the widget, in this particular case you are only using the voteAverage, so you don't really need to use all the movie object